app/uk/gov/hmrc/transitmovementsauditing/services/AuditService.scala

```diff
@@ -48,6 +49,11 @@ import scala.util.control.NonFatal
 @ImplementedBy(classOf[AuditServiceImpl])
 trait AuditService {
 
+  def sendMessageTypeEvent(auditType: AuditType, details: Details)(implicit
+    hc: HeaderCarrier
+  ): EitherT[Future, AuditError, Unit]
+
+  // TODO: still required??
```

Comment:
Good question, once Prashanth's stuff is in, if it's not needed, bin it.

Reply:
Removed.